test/src/kbasesearchengine/test/events/handler/WorkspaceEventHandlerTest.java

```diff
@@ -255,17 +255,17 @@ public void loadWithPathMaximalCopyRef() throws Exception {
                                 .withCreator("creator")
                                 .withCopied("7/8/9")
                                 .withCopySourceInaccessible(0L)
-                                .withInfo(objTuple(2, "objname", "sometype", "date", 3, "copier",
-                                        1, "wsname", "checksum", 44, Collections.emptyMap()))))));
-        
-        when(wscli.administer(argThat(new AdminGetWSInfoAnswerMatcher(1))))
-                .thenReturn(new UObject(wsTuple(1, "wsname", "username", "date", 7, "r", "n",
+                                .withInfo(objTuple(5, "objname", "sometype", "date", 6, "copier",
```

Comment:
What are the changes here for?

Reply:
Changes so that the mock for input "1/2/3;4/5/6" returns the WS info of "4/5/6".